game/client_types/autoplay.js

```diff
@@ -10,13 +10,13 @@
 
 module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
 
-    var channel = gameRoom.channel;
-    var node = gameRoom.node;
-    var ngc =  require('nodegame-client');
+    let channel = gameRoom.channel;
+    let node = gameRoom.node;
+    const ngc =  require('nodegame-client');
 
-    var game, stager;
+    var stager;
```

Comment:
Simply remove `var stager`.

Reply:
It is already defined as a function parameter, we overwrite it without redefining it.